packages/blockchain-link-types/src/blockbook.ts

```diff
@@ -7,44 +7,38 @@ import type {
     AccountInfoParams,
 } from './params';
 import type { AccountBalanceHistory, FiatRates, TokenStandard } from './common';
+import {
+    Vin,
+    Vout,
+    Utxo as BlockbookUtxo,
+    Tx as BlockbookTx,
+    TokenTransfer as BlockbookTokenTransfer,
+    Block as BlockbookBlock,
+    Token as BlockbookToken,
+    WsInfoRes,
+    WsBlockHashRes,
+    Address as BlockbookAddress,
+} from './blockbook-api';
+
+type Required<Type, Key extends keyof Type> = Type & {
+    [Property in Key]-?: Type[Property];
+};
+
+export type AccountUtxo = Required<BlockbookUtxo, 'address' | 'height' | 'value' | 'path'>[];
```

Comment:
```suggestion
type RequiredKey<M, K extends keyof M> = Omit<M, K> & Required<Pick<M, K>>;

export type AccountUtxo = RequiredKey<BlockbookUtxo, 'address' | 'height' | 'value' | 'path'>[];
```

To be replacable by `RequiredKey` from `type-utils` in the future